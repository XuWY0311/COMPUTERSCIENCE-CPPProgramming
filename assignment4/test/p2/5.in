100
-155 44 13 -160 155 288 -42 -39 123 -29 -22 -71 -101 -98 -2 -1 -8 -100 8 -80 52 77 -88 -75 -73 117 8 43 15 62 205 115 -80 75 -53 -78 91 -67 152 -41 -71 43 121 10 -237 177 57 27 -39 117 -123 -62 13 49 72 -161 89 55 62 105 -45 -24 77 -91 -16 -24 -40 -275 125 -143 42 -106 7 -55 -81 -130 225 26 61 103 -146 84 -76 37 82 100 -165 -59 -99 78 76 -49 -58 -17 -83 -13 -111 100 -62 -39