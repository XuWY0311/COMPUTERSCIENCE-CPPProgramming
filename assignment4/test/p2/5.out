{ -275, -237, -165, -161, -160, -155, -146, -143, -130, -123, -111, -106, -101, -100, -99, -98, -91, -88, -83, -81, -80, -80, -78, -76, -75, -73, -71, -71, -67, -62, -62, -59, -58, -55, -53, -49, -45, -42, -41, -40, -39, -39, -39, -29, -24, -24, -22, -17, -16, -13, -8, -2, -1, 7, 8, 8, 10, 13, 13, 15, 26, 27, 37, 42, 43, 43, 44, 49, 52, 55, 57, 61, 62, 62, 72, 75, 76, 77, 77, 78, 82, 84, 89, 91, 100, 100, 103, 105, 115, 117, 117, 121, 123, 125, 152, 155, 177, 205, 225, 288 }